dist/manifestor.js

```diff
@@ -2705,7 +2705,7 @@ var manifestor = function(options) {
       renderLayout(layout.intermediate(), true, endCallback);
```

Comment:
I'm confused. Is this just a hiccup that the Dist folder is being included? I must have built somehow.